MyCryptoHQ/MyCrypto
# ev/styledlint

To prevent https://mycryptohq.slack.com/archives/GB0GHM693/p1582188919076100 from happening again we:

1. Include stylelint and configure to use for `.tsx` files
2. Fix existing errors
3. Add task to `husky` pre-commit hook
4. Add  task to webpack-dev-server

### How to test
1. Clone the project.
2. Select a component that uses styled-components
3. Create a syntax error
4. See webpack emit associated errors
